src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -150,6 +154,16 @@ impl Sandbox for VirtSandbox {
             .await
             .context("setup device after start vm")?;
 
+        // get guest hook path
+        // TODO: maybe add a function for toml_config to get config from hypervisor name since there are multiple places do this
+        let hypervisor_name = &self.config.runtime.hypervisor_name;
```

Comment:
I think you don't need to put the `TomlConfig` inside the `VirtSandbox` struct cause you can just use the `self.hypervisor.hypervisor_config().await.security_info.guest_hook_path` to get that.

Reply:
Thanks @Tim-0731-Hzt , fixed.